__tests__/app.test.js

```diff
@@ -319,6 +319,24 @@ describe('app', () => {
                 });
         });
     });
+    describe('GET /api/users', () => {
+        test('Responds 200 and returns an array', () => {
+            return request(app).get('/api/users').expect(200).then((response) => {
+                const users = response.body.users
+                expect(Array.isArray(users)).toBe(true);
+            })
+        })
+        test('Users array items are objects with keys for username, name and avatar_url', () => {
+            return request(app).get('/api/users').expect(200).then((response) => {
```

Comment:
Don't forget to assert the length of the array is correct when using a forEach to stop false positives. 😄